.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### How to use
+Leave the headings unless they don't apply to your PR, replace the explanatory text with text describing your PR.
+
+#### Summary
+A one-line description of your change that will be extracted and added to the [project changelog](../data/changelog.txt).  
+The format is ```SUMMARY: Category "description"```  
+Category is one of Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N
+
+#### Purpose of change
+If theres an existing issue describing the problem this PR addresses or the feature it adds, please link it like #1234  
+If it *fully* resolves an issue, link it like ```Fixes #1234```  
+Even if the issue describes the problem, please provide a few-sentence summary here.  
+If there is no related issue, please describe the issue you are addressing, including how to trigger a bug if this is a bugfix.
+
+#### Describe the solution
+How does the feature work, or how does this fix a bug?  
+The easier you make your solution to understand, the faster it can get merged.
```

Comment:
Suggest: `The easier you make your solution to understand` to `The easier your solution is to understand`

Reply:
I prefer the active voice, even if their solution is difficult, this is their opportunity to *make* it easier to understand.